app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,45 @@
+class CommentsController < ApplicationController
+  before_action :authorize_user, only: [:edit, :update, :destroy]
```

Comment:
Here you call the `authorize_user` action but when you defined it at the bottom you used an exclamation mark in the name `authorize_user!` and this throws an error for the edit and destroy actions:
<img width="1105" alt="Screen Shot 2019-08-06 at 11 01 55 PM" src="https://user-images.githubusercontent.com/698811/62593930-43011a00-b89e-11e9-8642-89e1c8306994.png">

Reply:
Linter is added, comments controller authenticate_user method refined, we will open the review requests to development. The omniauth feature was told by another reviewer, we told them it is the topic of another milestone but they wanted us to add it. What should we do now should we merge it to development?

Reply:
Hi @YemreAybey, the idea is when I check that all the changes are completed you can merge them into `development` but for some reason Stickler isn't showing if there are lint errors or not. Did you turn on Stickler for this project in the Stickler page?